test/tokenbridge.test.ts

```diff
@@ -99,49 +101,82 @@ describe('TokenBridge', async function() {
   let usdc: Contract;
 
   before(async function () {
-
     // load L1 <--> L2 messaging contract
 
-    mockStarknetMessagingAddress = (await starknet.devnet.loadL1MessagingContract(networkUrl)).address;
+    mockStarknetMessagingAddress = (
+      await starknet.devnet.loadL1MessagingContract(networkUrl)
+    ).address;
 
     // L2 deployments
 
     l2user = await starknet.deployAccount("OpenZeppelin");
 
-    TokenBridgeL2 = await starknet.getContractFactory('rewaave/token_bridge');
+    TokenBridgeL2 = await starknet.getContractFactory("rewaave/token_bridge");
     tokenBridgeL2 = await TokenBridgeL2.deploy();
 
-    ProxyFactoryL2 = await starknet.getContractFactory('proxy');
-    proxyTokenBridgeL2 = await ProxyFactoryL2.deploy({proxy_admin: BigInt(l2user.starknetContract.address)});
-    proxyL2TokenDai = await ProxyFactoryL2.deploy({proxy_admin: BigInt(l2user.starknetContract.address)});
-    proxyL2TokenUsdc = await ProxyFactoryL2.deploy({proxy_admin: BigInt(l2user.starknetContract.address)});
+    ProxyFactoryL2 = await starknet.getContractFactory("proxy");
+    proxyTokenBridgeL2 = await ProxyFactoryL2.deploy({
+      proxy_admin: BigInt(l2user.starknetContract.address),
```

Comment:
I've just noticed a flaw here: the token bridge shouldn't be deployed by `l2user` but by another user named `l2deployer` for instance. We want `l2user` to be a random user that uses the contract. Otherwise, the test may reflect what's possible to do if you are the deployer of the contract, but not a simple user.

Reply:
https://github.com/aave-starknet-project/aave-starknet-bridge/projects/1#card-81479497

Reply:
IMO we need to have a separate PR to improve tests

Reply:
Yes this will be in a separate pr